### ISSUE TITLE
Named export 'asap' not found / NextJS

### ISSUE THREAD

**Issue by varunarora** [react-dnd/react-dnd#3383]
**Describe the bug**
Hi! I am a DnD newbie, and have been out of all of the details of the JavaScript/webpack moduling and tooling world for a while, so can't figure this out even though this might be really easy for some of you to figure out.

I am using NextJS (12.0.10) and React (17.0.2). I don't access to their internal webpack config. When trying to import the DnD `useDrag` export (version 15.1.0) for the first time, I see this error:

```
SyntaxError: Named export 'asap' not found. The requested module '@react-dnd/asap' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@react-dnd/asap';
const { asap } = pkg;
```

Begins with this call stack item:
```
dnd-core/dist/esm/classes/HandlerRegistryImpl.mjs (6)
```

**Reproduction**

1. `npx create-react-app`
2. cd `<project_name>`
3. `npm install --save react-dnd react-dnd-html5-backend`
4. In `pages/index.js`, import using `import { useDrag } from 'react-dnd'`

**Additional context**
A downgrade to v14 makes the issue go away.

**Comment by darthtrevino** [comment]
This is definitely something that's fixable, and I'll plan on verifying that those dependencies are ESM. 

I also added a Next.JS tester in the app here (https://github.com/react-dnd/react-dnd/pull/3384), and it appears to work fine as-is

**Comment by darthtrevino** [comment]
I've updated the asap dependency to export ESM; let me know if this issue continues

**Comment by varunarora** [comment]
Wow thank you @darthtrevino you are so quick and amazing!

**Comment by dustin-H** [comment]
Hi @darthtrevino,
thanks for fixing that issue.

It seems to be released in version `15.1.2`. (https://github.com/react-dnd/react-dnd/releases/tag/v15.1.2)
However, this versions hasn't been released on npm.

Could you please release it there? 😃 
Thanks! 🍻
